### ISSUE TITLE
binary overlay short-comings: those binaries are not running

### ISSUE THREAD

**Issue by dougbtv** [redhat-nfvpe/kube-ansible#86]
{see Tomo's notes}

**Comment by s1061123** [comment]
let me add a background for further guys:
Currently kube-centos-ansible uses `kubeadm` to deploy k8s. kubeadm uses hyperkube, k8s services in container.
API server, kubeproxy and other services are pulled from kubernetes repository as container image, hence current ansible playbook uses kubernetes repository's container, not own build.

I will add more information about it later.

**Comment by dougbtv** [comment]
Thanks for fleshing this out -- that's about what I thought you were driving at, I had some notes that you mentioned just about the same thing regarding my "how to compile kubernetes" blog article -- that because of kubeadm, it's not running those binaries. 

I'm going to change the title of this one.

**Comment by leifmadsen** [comment]
Added a comment on #66 that makes me wonder if this could be addressed during my own issues (#118, etc). Something to discuss anyways. I think as part of the work of `auto-kube-dev` in `kube-centos-ansible` that we could solve this as part of that.

**Comment by leifmadsen** [comment]
The next step here for me is to load the images and binaries in after the builder/sync job is completed. Once that is done, this issue should effectively be completed.

**Comment by leifmadsen** [comment]
@s1061123, @dougbtv, I think this issue is complete now that #159 has been resolved. We're now able to (via automation) build and consume artifacts from a k8s repo. I'm going to close this issue out, but feel free to reopen if you don't agree (or open another issue if we're missing something). Thanks!
